package.json

```diff
@@ -26,7 +26,7 @@
     "docusaurus-plugin-sass": "0.2.1",
     "docusaurus2-dotenv": "^1.4.0",
     "dotenv": "^8.2.0",
-    "protect-button": "^0.4.2",
+    "protect-button": "../protect-button/protect",
```

Comment:
this is only temporary for draft right?

Reply:
yeah sure, I was waiting for proper version published to npm. I'm pushing version with proper dependency soon